src/bubble.py

```diff
@@ -0,0 +1,46 @@
+"""Bubble sort algorithm."""
+
+
+def bubble_sort(l):
+    """Sort a list of numbers by bubble sort."""
+    if isinstance(l, list):
+        swap = 1
+        while swap:
+            swap = 0
+            for i in range(len(l) - 1):
+                if l[i] > l[i + 1]:
+                    tmp = l[i]
+                    l[i] = l[i + 1]
+                    l[i + 1] = tmp
```

Comment:
Use multiple assignment instead of writing 3 lines with an intermediate temporary variable